src/QsCompiler/Tests.Compiler/TestCases/ExecutionTests/LoggingBasedTests.qs

```diff
@@ -33,10 +33,12 @@ namespace Microsoft.Quantum.Testing.ExecutionTests {
             ULog("V1");
 
             within {
+                let dummy = 0;
```

Comment:
I'm not sure what this dummy variable is used to test. Can you please explain?

Reply:
It's a good idea to make sure the transformation that handles eliminating the apply-within statements deals appropriately with variable names. It is valid to defined a variable with the same name in both blocks, and correspondingly variables need to be renamed upon inclining the blocks.